docs/development.md

```diff
@@ -343,7 +343,7 @@ To run E2E locally, set `AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`, `AZURE_SUBSCRI
 ./scripts/ci-e2e.sh
 ```
 
-You can optionally set `AZURE_SSH_PUBLIC_KEY_FILE` to use your own SSH key.
+You can optionally set `SSH_KEY_FILE` to use your own SSH key.
```

Comment:
is it possible to keep the same name? it would be easier for current users to not have to set a new env var